Code/GraphMol/RWMol.cpp

```diff
@@ -267,6 +269,7 @@ void RWMol::removeAtom(Atom *atom) {
   boost::clear_vertex(vd, d_graph);
   // finally remove the vertex itself
   boost::remove_vertex(vd, d_graph);
+  delete atom;
```

Comment:
I think this is going to result in leaking the memory for the bonds to the atom. The call to boost::clear_vertex() will remove the edges from the graph, but won't delete the bonds.

Reply:
This is handled in the removeBonds function I think:

```
for (auto &nbr : nbrs) {
    removeBond(nbr.first, nbr.second);
  }
```

Reply:
Sorry, I missed that. You're right.